src/material-examples/table-expandable-rows/table-expandable-rows-example.html

```diff
@@ -29,7 +29,7 @@
   <tr mat-row *matRowDef="let element; columns: columnsToDisplay;"
       class="example-element-row"
       [class.example-expanded-row]="expandedElement === element"
-      (click)="expandedElement = element">
+      (click)="expandedElement === element ? expandedElement = null : expandedElement = element;">
```

Comment:
This would be slightly shorter as `expandedElement = expandedElement === element ? null : element`.